src/Payments/CardPayment.res

```diff
@@ -253,13 +270,32 @@ let make = (
                   )}
                   appearance=config.appearance
                   type_="tel"
-                  className="tracking-widest w-full"
+                  className={`tracking-widest w-full ${innerLayout === Compressed &&
```

Comment:
Please move this condition outside as well

Reply:
Updated